js/main.js

```diff
@@ -87,55 +110,106 @@ var renderMapPin = function (template, ad) {
   return mapElement;
 };
 
-var getDeclOfNumb = function (number, titles) {
-  var cases = [2, 0, 1, 1, 1, 2];
+var getAddressMapPinMainStr = function (isDisable) {
+  var сalculatedX = parseInt(mapPinMainNode.style.left, 10) + MAIN_PIN_SIZES.active.width / 2;
+  var сalculatedHeight = isDisable ? MAIN_PIN_SIZES.active.height / 2 : MAIN_PIN_SIZES.active.height;
+  var сalculatedY = parseInt(mapPinMainNode.style.top, 10) + сalculatedHeight;
 
-  return titles[(number % 100 > 4 && number % 100 < 20) ? 2 : cases[(number % 10 < 5) ? number % 10 : 5]];
+  return Math.round(сalculatedX) + ', ' + Math.round(сalculatedY);
 };
 
-var renderPopup = function (template, ad) {
-  var popupElement = template.cloneNode(true);
-  var popupElementFeatures = popupElement.querySelector('.popup__features').children;
-  var popupElementPhoto = popupElement.querySelector('.popup__photo');
-
-  popupElement.querySelector('.popup__avatar').src = ad.author.avatar;
-  popupElement.querySelector('.popup__title').textContent = ad.offer.title;
-  popupElement.querySelector('.popup__text--address').textContent = ad.offer.address;
-  popupElement.querySelector('.popup__text--price').textContent = ad.offer.price + ' ₽/ночь';
-  popupElement.querySelector('.popup__type').textContent = TYPES_MAP[ad.offer.type];
-  popupElement.querySelector('.popup__text--time').textContent = 'Заезд после ' + ad.offer.checkin + ', выезд до ' + ad.offer.checkout;
-  popupElement.querySelector('.popup__description').textContent = ad.offer.description;
-
-  popupElement.querySelector('.popup__text--capacity').textContent = ad.offer.rooms +
-    getDeclOfNumb(ad.offer.rooms, [' комната', ' комнаты', ' комнат']) + ' для ' +
-    ad.offer.guests + getDeclOfNumb(ad.offer.guests, [' гостя', ' гостей', ' гостей']);
-
-  for (var i = 0; i < popupElementFeatures.length - ad.offer.features.length; i++) {
-    popupElementFeatures[popupElementFeatures.length - 1 - i].style.display = 'none';
+var toggleDisabledOnFormNodes = function (isDisable) {
+  var pointerEventsValue = isDisable === true ? 'none' : 'auto';
```

Comment:
`isDisable` содержит булево значение, потому нет смысла дополнительной проверке
```suggestion
  var pointerEventsValue = isDisable ? 'none' : 'auto';
```